src/ForkTsCheckerWebpackPluginOptions.ts

```diff
@@ -7,6 +7,7 @@ import LoggerOptions from './logger/LoggerOptions';
 interface ForkTsCheckerWebpackPluginOptions {
   async?: boolean;
   typescript?: TypeScriptReporterOptions;
+  typescriptPath?: string;
```

Comment:
please move it to the TypeScriptReporterOptions instead of top-level options :)

Reply:
done